front/app/containers/Admin/users/ChangeSeatModal.tsx

```diff
@@ -64,6 +61,7 @@ const ChangeSeatModal = ({
   userToChangeSeat,
   toggleAdmin,
 }: Props) => {
+  const [showSuccess, setShowSuccess] = useState(false);
   const isUserAdmin = isAdmin({ data: userToChangeSeat });
```

Comment:
Could you change this var name? I again thought it was about the signed-in user :-) `isUserToChangeSeatAdmin`?